updated files for assignment

laflad opened the pull request.

laflad committed:
updated files for assignment

kearlyaa commented on lesson02/assignment/css/style.scss:
```diff
@@ -3,4 +3,89 @@
 @import 'settings.variables';
 @import 'settings.responsive';
 @import 'components.content';
+@import 'settings.font';
```
Looks like a typo.  I think font needs an 's' at the end to match the _settings.fonts.scss file.

kearlyaa commented on lesson02/assignment/index.html:
```diff
@@ -9,7 +9,40 @@
+        </div>
+
+    </div>
     </body>
```
Looks like this closing body tag should go after the closing main tag.

laflad commented:
Hi Aaron,

Thank you very much for the feedback and guidance. Sorry it took so long to get back to you. I had to travel to film a wedding and work got slammed again which is largely why I was hoping I could pause the course until winter where it calms down a little. But I’m slowly making progress.

I went ahead and fixed the html, fixed the typo in my scss and added comments. I did another pull request.

I have to say that it’s refreshing that you’re looking at my code. The first two classes felt like I never got feedback and never knew if I was doing things correctly so I appreciate the guidance. This is refreshing!

New Pull Request <https://github.com/laflad/html300-v2/pull/3> < for the fixes from your feedback

Thank you!!

David




> On Nov 9, 2022, at 9:17 AM, Aaron Kearly ***@***.***> wrote:
> 
> 
> @kearlyaa commented on this pull request.
> 
> In lesson02/assignment/index.html <https://github.com/laflad/html300-v2/pull/2#discussion_r1018214645>:
> 
> > +            </div>
> +            <div class="lineitem">
> +                <p class="category">School: </p>
> +                <p class="detail"> University of Washington</p>
> +            </div>
> +            <div class="lineitem">
> +                <p class="category">Major: </p>
> +                <p class="detail">Marketing</p>
> +            </div>
> +            <div class="lineitem">
> +                <p class="link"><svg enable-background="new 0 0 32 32" height="32px" id="Layer_1" version="1.0" viewBox="0 0 32 32" width="32px" xml:space="preserve" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g><path d="M32,30c0,1.104-0.896,2-2,2H2c-1.104,0-2-0.896-2-2V2c0-1.104,0.896-2,2-2h28c1.104,0,2,0.896,2,2V30z" fill="#007BB5"/><g><rect fill="#FFFFFF" height="14" width="4" x="7" y="11"/><path d="M20.499,11c-2.791,0-3.271,1.018-3.499,2v-2h-4v14h4v-8c0-1.297,0.703-2,2-2c1.266,0,2,0.688,2,2v8h4v-7    C25,14,24.479,11,20.499,11z" fill="#FFFFFF"/><circle cx="9" cy="8" fill="#FFFFFF" r="2"/></g></g><g/><g/><g/><g/><g/><g/></svg>
> +                paolo.linkedinprofile.com</p>
> +            </div>
> +        </div>
> +
> +    </div>
>      </body>
> Looks like this closing body tag should go after the closing main tag.
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/laflad/html300-v2/pull/2#pullrequestreview-1174457471>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ACEVBJQ7P6GZGXK3YLYKUHLWHPMCPANCNFSM6AAAAAAR3UKSDM>.
> You are receiving this because you authored the thread.
>
